docroot/themes/humsci/su_humsci_theme/patterns/molecules/generic-three-column/scss/generic-three-column.scss

```diff
@@ -34,4 +34,9 @@
     }
   }
 
+  &[class*="1-5-1"] {
```

Comment:
Why the wildcard class selector? There are only two classes with the convention you want. Why not just name them here? I would suspect that explicitly declaring the classes to act upon is the more performant (but less sexy) code.

Reply:
you make a good point. i'm on board :boat: